src/components/CommentList/CommentList.tsx

```diff
@@ -1,5 +1,19 @@
 import React from 'react';
+import { CommentInfo } from '../CommentInfo';
+import { Comment } from '../../../types/Comment';
+import './CommentList.scss';
 
-export const CommentList: React.FC = () => (
-  <>Put the list here</>
+export interface CommentListProps {
+  comments: Comment[];
+}
+
+export const CommentList: React.FC<CommentListProps> = ({ comments }) => (
+  <div className="CommentList">
+    {comments.map(comment => (
+      <CommentInfo
```

Comment:
Unnecessary line break; the tag only has 2 properties, single line is acceptable